File: sdkwork-backend-sdk-go/types/usage_record_vo.go
package types

// AI模型使用记录VO，用于记录各种AI模型调用的使用情况
type UsageRecordVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	UserId int `json:"userId"`
	RequestId string `json:"requestId"`
	ModelName string `json:"modelName"`
	Channel string `json:"channel"`
	UsageType string `json:"usageType"`
	BillingType string `json:"billingType"`
	PromptTokens int `json:"promptTokens"`
	CompletionTokens int `json:"completionTokens"`
	CachedTokens int `json:"cachedTokens"`
	TotalTokens int `json:"totalTokens"`
	CallCount int `json:"callCount"`
	ProcessingTime int `json:"processingTime"`
	ImageCount int `json:"imageCount"`
	ImageSize string `json:"imageSize"`
	VideoDuration int `json:"videoDuration"`
	AudioDuration int `json:"audioDuration"`
	Cost float64 `json:"cost"`
	CurrencyCode string `json:"currencyCode"`
	Status string `json:"status"`
	RequestTime string `json:"requestTime"`
	ResponseTime string `json:"responseTime"`
	PromptId int `json:"promptId"`
	ToolId int `json:"toolId"`
}

File: sdkwork-backend-sdk-csharp/Models/UsageRecordVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class UsageRecordVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public int? UserId { get; set; }
        public string? RequestId { get; set; }
        public string? ModelName { get; set; }
        public string? Channel { get; set; }
        public string? UsageType { get; set; }
        public string? BillingType { get; set; }
        public int? PromptTokens { get; set; }
        public int? CompletionTokens { get; set; }
        public int? CachedTokens { get; set; }
        public int? TotalTokens { get; set; }
        public int? CallCount { get; set; }
        public int? ProcessingTime { get; set; }
        public int? ImageCount { get; set; }
        public string? ImageSize { get; set; }
        public int? VideoDuration { get; set; }
        public int? AudioDuration { get; set; }
        public double? Cost { get; set; }
        public string? CurrencyCode { get; set; }
        public string? Status { get; set; }
        public string? RequestTime { get; set; }
        public string? ResponseTime { get; set; }
        public int? PromptId { get; set; }
        public int? ToolId { get; set; }
    }
}
